### DISCUSSION TITLE
Suggestion for evaluation

### DISCUSSION THREAD

**Discussion by wang975679801** [PaddlePaddle/PaddleSpatial#134] [BaiduKDDCup2022] [answered]
The online testing loads the model files repeatedly which costs too much time. Could we just load model files only once? I think this can reduce the computational stress significantly.

**Comment by ramber1836** [comment, marked as answer]
Thanks for your suggestion!
Actually, the online test does not constrain the way of loading models, it is the competitor's decision.
Though the evaluation script invokes the forecast interface separately, the competitors can also load their model files once, say, load the model at the first run and access the loaded model afterward till the end of the evaluation.

**Reply by gengdd** [reply]
We have 195 test files and it is time consuming to reload the model each time a test file is read. Can we try to adjust to read the model first, and then traverse 195 test files, which can reduce the computational pressure, but it will be more troublesome in interface design.
